migrations/01_create_peep_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peep(id SERIAL PRIMARY KEY, message VARCHAR(140), posting_date DATE NOT NULL DEFAULT CURRENT_TIMESTAMP);
```

Comment:
Not entirely necessary but could have another file here for the command to create a table in the test database.